src/pages/ClaimsPage/utils.ts

```diff
@@ -33,6 +33,30 @@ export function formatDistributionDates(claims: QueryClaim[]) {
   )} ${endDate.getDate()} ${endDate.getFullYear()}`;
 }
 
+/**
+ * Generates a text representation showing the date range and number of
+ * distributions in the claims group. This function only relies on the
+ * Distribution's creation date in the database since deleted users
+ * don't have access to epoch data
+ * @param claims - a group of claims that can be claimed togoether in one tx.
+ */
+export function formatDeletedDistributionDates(claims: QueryClaim[]) {
```

Comment:
I think this specific one is a fairly significant departure in terms of semantics from the non-deleted case. I'm afraid it might not be worth the lift.

Reply:
sounds good - if not easy then not worth the lift!